schema-inspector.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
```

Comment:
Normally packages use an index like this
```
module.exports = require('./lib')
```
Lib directory contains source code and exports in an index file